java/ql/test/library-tests/frameworks/spring/util/test.ql

```diff
@@ -0,0 +1,54 @@
+import java
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.dataflow.ExternalFlow
+import semmle.code.java.dataflow.TaintTracking
+import TestUtilities.InlineExpectationsTest
+import semmle.code.java.frameworks.spring.Spring
```

Comment:
Remove. If the test requires this to pass then it just shows that bidirectional import is missing.
```suggestion
```